examples/circlegraph/src/standalone.ts

```diff
@@ -77,11 +77,15 @@ export default async function runCircleGraph() {
         graph.children.push(...newElements);
     }
 
+    let viewport = initialViewport;
+    window.addEventListener('resize', async () => {
+        viewport = await modelSource.getViewport();
+    });
+
     // Run
     modelSource.setModel(graph);
 
     async function createNode(point?: Point) {
-        const viewport = await modelSource.getViewport();
```

Comment:
The example now starts with all nodes in the top left corner:
<img width="1126" alt="Screenshot 2024-02-16 at 11 13 14" src="https://github.com/eclipse-sprotty/sprotty/assets/4067210/80bd00e8-4b6b-4f1a-8e32-2f4124c739a0">

Reply:
It starts like that since https://github.com/eclipse-sprotty/sprotty/commit/b0e9d60e9c9ab7d302f84f55eef227828ba7d68c